package.json

```diff
@@ -4,16 +4,27 @@
   "description": "An app to manage the awesomeness that is Friday Projects",
   "main": "index.js",
   "scripts": {
-    "test": "tape 'tests/**/*.js'"
+    "test": "xo && tape 'tests/**/*.js'"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/kahikatea-2016/friday-projects.git"
   },
+  "xo": {
+    "rules": {
+      "space-before-function-paren": ["error", {
+  			"anonymous": "always",
```

Comment:
It looks like the indenting got all weird for some reason. Can you also make sure we don't have tabs in there.